src/community/resources/index.md.njk

```diff
@@ -0,0 +1,103 @@
+---
+title: Resources
+description: Useful resources and community projects
+section: Community
+layout: layout-pane.njk
+order: 1
+status: Experimental
+---
+
+A curated list of community developed resources that can help you use the GOV.UK Design System in your chosen design tool or development environment. 
```

Comment:
“”
```suggestion
You can use these resources and tools to help you use the GOV.UK Design System in your design tool or development environment. 
```

We try to avoid ‘pages that talk about themselves’. Could you tweak the first paragraph to talk to the user about the need the page meets? Suggestion above.